aexpect/client.py

```diff
@@ -1223,16 +1228,22 @@ def cmd_output(self, cmd, timeout=60, internal_timeout=None,
             raise ShellTimeoutError(cmd, output) from error
         except ExpectProcessTerminatedError as error:
             output = self.remove_command_echo(error.output, cmd)
-            raise ShellProcessTerminatedError(cmd, error.status, output) from error
+            raise ShellProcessTerminatedError(
+                cmd, error.status, output) from error
         except ExpectError as error:
             output = self.remove_command_echo(error.output, cmd)
             raise ShellError(cmd, output) from error
 
         # Remove the echoed command and the final shell prompt
+        if strip_console_codes:
+            # Removing the escape sequence
+            output_no_escape = astring.strip_console_codes(out)
+            return self.remove_last_nonempty_line(self.remove_command_echo(output_no_escape,
+                                                                           cmd))
```

Comment:
How about a much simpler
```python
out = astring.strip_console_codes(out) if strip_console_codes else out
```
?

Reply:
Well that's an extra assignment in case `strip_console_codes` is `False`. But I do like the idea of having just one return as they are mainly the same. How about:

```
if strip_console_codes:
    out = astring.strip_console_codes(out)
```

Reply:
It is a noop assignment and the above is a very pythonic way to perform such variable assignments but of course your suggestion is also fine.